src/PluginEnumerateDevices.swift

```diff
@@ -8,7 +8,7 @@ import AVFoundation
 
 
 class PluginEnumerateDevices {
-	class func call(callback: (data: NSDictionary) -> Void) {
+	class func call(_ callback: (_ data: NSDictionary) -> Void) {
```

Comment:
this was intriguing, so I did a bit of research: http://stackoverflow.com/questions/39630941/ios-swift-3-parameters-have-underscore-in-front

apparently in swift (3) by default all arguments are keyword-arguments and so the names must be specified when a function is called. Adding the `_` basically turns it into a positional parameter that doesn't need to be labelled on the calling side (it still uses the name to be referred to in the function body).

In swift 2 apparently the first function argument was *always* positional and the remaining keyword-parameters (by default at least).